src/docfx/Models/BuildJsonConfig.cs

```diff
@@ -123,5 +123,8 @@ public class BuildJsonConfig
 
         [JsonProperty("customLinkResolver")]
         public string CustomLinkResolver { get; set; }
+
+        [JsonProperty("versions")]
+        public Dictionary<string, VersionInfo> Versions { get; set; }
```

Comment:
>Versions [](start = 47, length = 8)

what's the relationship between the version key and FileMappingItem.VersionName?